avalanche/benchmarks/utils/detection_dataset.py

```diff
@@ -359,43 +300,6 @@ def _detection_class_mapping_transform(class_mapping, example_target_dict):
     return example_target_dict
 
 
-@overload
-def detection_subset(
-    dataset: SupervisedDetectionDataset,
-    indices: Optional[Sequence[int]] = None,
-    *,
-    class_mapping: Optional[Sequence[int]] = None,
-    transform: Optional[XTransform] = None,
-    target_transform: Optional[YTransform] = None,
-    transform_groups: Optional[Mapping[str, 
-                                       Tuple[XTransform, YTransform]]] = None,
-    initial_transform_group: Optional[str] = None,
-    task_labels: Optional[Union[int, Sequence[int]]] = None,
-    targets: Optional[Sequence[TTargetType]] = None,
-    collate_fn: Optional[Callable[[List], Any]] = None
-) -> SupervisedDetectionDataset:
-    ...
-
-
-@overload
-def detection_subset(
-    dataset: SupportedDetectionDataset,
-    indices: Optional[Sequence[int]] = None,
-    *,
-    class_mapping: Optional[Sequence[int]] = None,
-    transform: Optional[XTransform] = None,
-    target_transform: Optional[YTransform] = None,
-    transform_groups: Optional[Mapping[str, 
-                                       Tuple[XTransform, YTransform]]] = None,
-    initial_transform_group: Optional[str] = None,
-    task_labels: Union[int, Sequence[int]],
-    targets: Sequence[TTargetType],
-    collate_fn: Optional[Callable[[List], Any]] = None
-) -> SupervisedDetectionDataset:
-    ...
-
-
-@overload
 def detection_subset(
     dataset: SupportedDetectionDataset,
```

Comment:
do we need this? DetectionDataset should be enough. We have the others for classificaiton mostly to keep it compatible with the old API.